compose/cli/main.py

```diff
@@ -720,14 +721,17 @@ def run(self, options):
         running. If you do not want to start linked services, use
         `docker-compose run --no-deps SERVICE COMMAND [ARGS...]`.
 
-        Usage: run [options] [-v VOLUME...] [-p PORT...] [-e KEY=VAL...] SERVICE [COMMAND] [ARGS...]
+        Usage:
+            run [options] [-v VOLUME...] [-p PORT...] [-e KEY=VAL...] [-l KEY=VALUE...]
+                [--label KEY=VALUE...] SERVICE [COMMAND] [ARGS...]
```

Comment:
You don't need the two versions of the option in the `Usage`, only in the options list below. I would remove the long form for consistency